src/Kernel/CompiledMethod.class.st

```diff
@@ -124,13 +124,7 @@ CompiledMethod class >> handleFailingNewMethod: numberOfBytes header: headerWord
 CompiledMethod class >> headerFlagForEncoder: anEncoderClass [
 	anEncoderClass == PrimaryBytecodeSetEncoderClass ifTrue: [ ^ 0 ].
 	anEncoderClass == SecondaryBytecodeSetEncoderClass ifTrue: [ ^ SmallInteger minVal ].
```

Comment:
It should be simple "anEncoderClass headerFlagForMethods"

Reply:
I was thinking of that, but wanted to not do it as part of the cleanup as it does change the semantics: the number now is not defined by the bytecode set, but by it being registered primary or secondary.